css/style.css

```
@@ -36,6 +36,19 @@ body {
     margin-bottom: 10px;
 }
 
+
+
+
+
+
+.site-footer {
```

@K-YYY 39-43行目は空行をあけるのではなく、コメントで見出しを追加するといいかも！
```suggestion
/* フッター */
.site-footer {
```

この機会に、ヘッダーのところにもコメントで見出しつけてしまいましょう！👍

参考までに、CSSのコメントについて記事貼っておきます！
https://www.sejuku.net/blog/72042